packages/boxel-ui/test-app/tests/integration/components/email-input-test.gts

```diff
@@ -0,0 +1,151 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+import {
+  fillIn,
+  render,
+  settled,
+  triggerEvent,
+  typeIn,
+} from '@ember/test-helpers';
+
+import { EmailInput } from '@cardstack/boxel-ui/components';
+
+async function waitForDebounce() {
+  await new Promise<void>((resolve) => setTimeout(resolve, 350));
+  await settled();
```

Comment:
Maybe we should have our own debounce that wraps the lodash one, because if we wire it into the Ember test waiter system, we won’t need things like this.

Reply:
Ended up using `debounce` from `'@ember/runloop'` instead of the `lodash` one. This one works with the built-in test helpers. https://guides.emberjs.com/release/testing/testing-components/#toc_waiting-on-asynchronous-behavior